src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -16,6 +25,8 @@
         {% elif size %}
             ({{size|readable_size}})
         {% endif %}
+        <br>
```

Comment:
Again: remove BR's and just separate elements with Spans so you can actually control the spacing.
Surround elements that need to be next to each other with another span (like the red dot and the 'Nieuw' text.